modules/post-installation-adding-nutanix-failure-domains-control-planes.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-adding-nutanix-failure-domains-control-planes_{context}"]
+= Distributing control planes across failure domains
+
+You distribute control planes across Nutanix failure domains by modifying the control plane machine set custom resource (CR).
+
+.Prerequisites
+
+* You have configured the failure domains in the cluster's Infrastructure custom resource (CR).
+* The control plane machine set custom resource (CR) is in active state.
```

Comment:
```suggestion
* The control plane machine set custom resource definition (CRD) is in active state.
```

Reply:
I'm not familiar with a CR being in _active state_. Would a typical user know how to determine this?

Reply:
Adding the correct article. Left CR.